### DISCUSSION TITLE
Yaml/PHP for content types?

### DISCUSSION THREAD

**Discussion by SuperFlySly** [sulu/sulu#7539] [Ideas]
Hi guys, would it also be possible to use yaml or PHP for the definition of content types?
And where should I start if I want to program my own implementation for this?

**Comment by alexander-schranz** [comment]
@SuperFlySly there is no support for yaml. But you can decorate form metadata loader service or best is to use formmetadatavisitors to only add the part which is dynamic via PHP.

**Comment by mamazu** [comment]
Cough cough #7218 

But in general, it would be great if we could move all configuration to Symfony config and we could use yaml and php as well.
